src/definitions.jl

```diff
@@ -271,7 +271,15 @@ summary(p::ScaledPlan) = string(p.scale, " * ", summary(p.p))
 *(p::Plan, I::UniformScaling) = ScaledPlan(p, I.λ)
 
 # Normalization for ifft, given unscaled bfft, is 1/prod(dimensions)
-normalization(::Type{T}, sz, region) where T = one(T) / Int(prod(sz[r] for r in region))::Int
```

Comment:
Seems simpler to just add `init=1` here?   `sz[r]` will do a bounds check, and this function is unlikely to be performance-critical.

Reply:
I agree. The original suggestion seems to have been a performance-centric one, but this time saved may be inconsequential.

Reply:
Updated